data_resource/config/configuration.py

```diff
@@ -204,7 +226,11 @@ def get_config(config_type: str):
 
         config_type = config_type.upper()
         if config_type == "TEST":
-            return TestConfig()
+            is_jenkins = bool(int(os.getenv("IS_JENKINS_TEST", "0")))
```

Comment:
Would you prefer 

```python
if config_type == "TEST":
    return TestConfig()
elif config_type == "JENKINS":
    return JenkinsConfig()
```

 and then remove the need for the extra env var `IS_JENKINS_TEST`?